src/api-reference/invoice/purchase-request-endpoints.markdown

```diff
@@ -0,0 +1,250 @@
+                  "vendorAddressCode" : "ADDR1",
+                  "description" : "monitor",
+                  "quantity" : "20",
+                  "unitPrice" : "154.4",
+                  "receiptType" : "QUANTITY_RECEIPT",
+                  "neededByDate": "2018-06-28",
+                  "uomCode" : "DA",
+                  "shipping" : "13.5",
+                  "tax" : "11",
+                  "supplierPartId" : "DAQT1",
+                  "url" :[
+                      "http://officesupplies.com/monitor"
+                  ],
+                  "notesToVendor" : "Phase 1 request monitor",
+                  "comments" : "Phase 1 request for new employees for monitor",
+                  "custom2" : "LGVT1"
+              },
+              {
+                  "purchaseType" : "GOODS",
+                  "vendorCode" :"VEN1",
+                  "vendorAddressCode" : "ADDR1",
+                  "description" : "office chair",
+                  "quantity" : "20",
+                  "unitPrice" : "346.2",
+                  "receiptType" : "NONE",
+                  "neededByDate": "2018-06-28",
+                  "uomCode" : "DA",
+                  "shipping" : "15",
+                  "tax" : "17.5",
+                  "supplierPartId" : "DAQT2",
+                  "url" :[
+                      "http://officesupplies.com/officechair"
+                  ],
+                  "notesToVendor" : "Phase 1 request office chair",
+                  "comments" : "Phase 1 request for new employees for office chair",
+                  "custom3" : "DEPT",
+                  "custom4" : "SALES"
+              }
+          ]
+  }'
+  ```
+    
+### Response  
+
+[Response schema](#create_purchase_request_schema-response)  
+
+
+- Example - Output
+ ```json
+     {
+        "id" : "b1e22581-ff4a-48e9-981b-2f5065579096",
+        "uri": "http://us.api.concursolutions.com/purchaserequest/v1/purchaserequests/b1e22581-ff4a-48e9-981b-2f5065579096?mode=COMPACT"
+     }
+ ```
+
+## <a name="get"></a>Get details of a purchase request
+
+    GET  /purchaserequest/v1/purchaserequests/{id}?mode=COMPACT
+
+Gets purchase request details. Currently only supported mode is COMPACT which returns basic info about the purchase request along with any exceptions if present.
+
+### <a name="8"></a>Parameters  
+
+|Name | Type | Format | Description
+|-----|------|--------|------------
+|`X_AUTH_TOKEN`|`string`|`header`|**Required**: Bearer Token that identifies the caller. This is the company JWT
+|`Content-Type`|`string`|`header`|**Required**: application/json
+|`mode`|`string`|`parameter`|**Required**: Specifies mode for get purchase request details. Currently supported mode COMPACT
+
+### Input  
+
+None
+
+### Response
+
+
+[Schema](#get_purchase_request_schema_response)
+
+- Example - Output
+ ```json
+     {
+        "purchaseRequestId" : "de9c0894-b807-6943-8e3f-49a707da3456",
+        "purchaseRequestNumber" : "PR100000",
```

Comment:
our PR number never starts with alphas, so we can just have 1000

Reply:
Fixed in f847a0bc